crawler/file_handler.py

```
@@ -73,20 +74,142 @@ def _extract_pdf(path: Path) -> tuple[str, str]:
+    prvtext = ""
     try:
         import olefile
 
         ole = olefile.OleFileIO(str(path))
         if ole.exists("PrvText"):
             raw = ole.openstream("PrvText").read()
-            return raw.decode("utf-16-le", errors="ignore").strip(), "olefile"
-        log.warning(f"HWP PrvText 스트림 없음: {path.name}")
-        return "", "olefile"
+            prvtext = raw.decode("utf-16-le", errors="ignore").strip()
     except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)